packages/sil-governance/src/routes/create-proposal/components/FormOptions/ReleaseGuaranteeFunds.tsx

```diff
@@ -2,21 +2,27 @@ import { FormApi } from "final-form";
 import { Block, TextFieldForm, Typography } from "medulas-react-components";
 import React from "react";
 
+const AMOUNT_LABEL = "Amount";
+export const RELEASE_QUANTITY_FIELD = "Quantity";
+const RELEASE_QUANTITY_PLACEHOLDER = "Quantity";
+
 interface Props {
   readonly form: FormApi;
 }
 
 const ReleaseGuaranteeFunds = ({ form }: Props): JSX.Element => {
   return (
-    <Block width="100%" display="flex" alignItems="center" marginTop={2} marginBottom={2}>
-      <Typography>Custom Field for ReleaseGuaranteeFunds </Typography>
-      <TextFieldForm
-        name="ReleaseGuaranteeFunds"
-        form={form}
-        placeholder="ReleaseGuaranteeFunds"
-        fullWidth
-        margin="none"
-      />
+    <Block marginTop={2} display="flex" alignItems="center">
+      <Typography>{AMOUNT_LABEL}</Typography>
+      <Block marginLeft={2}>
+        <TextFieldForm
+          name={RELEASE_QUANTITY_FIELD}
+          form={form}
+          type="number"
+          placeholder={RELEASE_QUANTITY_PLACEHOLDER}
+          margin="none"
+        />
+      </Block>
```

Comment:
Can we build something like the input in the wallet?

<img width="520" alt="Bildschirmfoto 2019-09-10 um 08 51 28" src="https://user-images.githubusercontent.com/2603011/64590612-39d90080-d3a8-11e9-98aa-24628bccd8ec.png">

The tickers can then be the ones available in the guarantee Fund account

```ts
// TODO: Calculate dynamically from guaranteeFundEscrowId using IOV-Core
const guaranteeFundAddress = "tiov170qvwm0tscn5mza3vmaerkzqllvwc3kycrz6kr" as Address;

const account = await connection.getAccount({ address: guaranteeFundAddress })
const tickers = (account ? account.balance.map(balance => balance.tokenTicker) : new Array<TokenTicker>());
```